metadata_mapper/validate_mapping.py

```diff
@@ -180,11 +191,11 @@ def get_couch_db_data(collection_id: int,
             "_list/has_field_value/by_provider_name_wdoc" \
             f"?key=\"{collection_id}\"&field={field_name}&limit=1000"
       
-      response = requests.get(url)
+      response = requests.get(url, verify=False)
```

Comment:
afaik this should no longer be necessary? I thought @barbarahui fixed this.

Reply:
Yeah, the ssl cert is renewed and valid. We should remove `verify=False` from this line. 

I think we also need to add `certifi` to `requirements.txt` in order to avoid a separate error being thrown by `requests` though. See: https://stackoverflow.com/questions/22027418/openssl-python-requests-error-certificate-verify-failed